configs/ruby/GPU_VIPER.py

```diff
@@ -669,23 +669,51 @@ def construct_gpudirs(options, system, ruby_system, network):
 
         # Create memory controllers too
         mem_type = ObjectList.mem_list.get(options.dgpu_mem_type)
-        dram_intf = MemConfig.create_mem_intf(
-            mem_type,
-            gpu_mem_range,
-            i,
-            int(math.log(options.dgpu_num_dirs, 2)),
-            options.cacheline_size,
-            xor_low_bit,
-        )
+
+        if options.hbm_ctrl:
```

Comment:
please add a comment about why we are adding 2 controllers here -- it's obvious to you, but less obvious to others trying to use this